glide-core/src/client/value_conversion.rs

```diff
@@ -182,6 +183,38 @@ pub(crate) fn convert_to_expected_type(
             )
                 .into()),
         },
+        // command returns nil or an array of 2 elements, where the second element is a map represented by a 2D array
+        // we convert that second element to a map as we do in `MapOfStringToDouble`
+        /*
+        127.0.0.1:6379> zmpop 1 z1 min count 10
+        1) "z1"
+        2) 1) 1) "2"
+              2) (double) 2
+           2) 1) "3"
+              2) (double) 3
+         */
+        ExpectedReturnType::ZMPopReturnType => match value {
+            Value::Nil => Ok(value),
```

Comment:
Should there be another conversion for Null as that is supported in RESP3?

Reply:
null supported by RESP 3 and 2 both. Statement on line 197 says that we do nothing with it if received (no conversion).

Reply:
null is usually returned when the blocking command times out - no data is returned